README.md

```diff
@@ -1,2 +1,454 @@
+
+The use case of "pairing" two DIDs by delegating all current and future resources is not uncommon when a user would like to use multiple devices as "root", but does not have access to all of them directly at all times.
+
+The format for this scheme is as follows:
+
+``` abnf
+ownershipscheme = "my:" kind ["@" did]
+kind = "*" / <scheme> 
+```
+
+The wildcard `*` resource MUST be taken to mean "everything" (all resources of all types).
+
+A "sub-scheme" MAY be used to delegate some of that scheme controlled by parenthood. For example `my:dns` delegates access to all DNS records (`my:dns:*` works equally well since wildcards are also part of the DNS URI). `my:mailto` selects all owned email addresses controlled by this user.
+
+Re-delegating these to further DIDs in a chain MUST address the specific parent DID that owns that resource separated by an `@`. For instance: `my:*@did:key:z6MkiTBz1ymuepAQ4HEHYSF1H8quG5GLVVQR3djdX3mDooWp` selects all resources originating from the specified DID, and `my:mailto@did:key:z6MkiTBz1ymuepAQ4HEHYSF1H8quG5GLVVQR3djdX3mDooWp` selects email addresses from the DID. 
+
+### 4.1.2 Action
+
+The action for `my:*` MUST be `*`.
+
+``` json
+{"with": "my:*", "can": "*"}
+```
+
+For `my` capabilities scoped to some scheme, the action MUST be one normally associated with that resource. 
+
+``` json
+{"with": "my:dns", "can": "crud/UPDATE"}
+```
+
+## 4.2 UCAN Addressing
+
+### 4.2.1 `prf` Scheme
+
+The `prf` URI scheme defines addressing for UCANs and their fields.
+
+``` abnf
+prf = "prf" selector
+selector = "*" / 1*DIGIT
+```
+
+`prf:*` represents all of the UCANs in the current proof scope. The witnesses in the `prf` field may be referenced by their index, starting at 0: `prf:0`
+
+### 4.2.2 `prf` Actions
+
+The `prf` scheme MUST accept the following action: `ucan/DELEGATE`. This re-delegates all of the capabilities in the selected witness(es).
+
+# 5. Validation
+
+Each capability has its own semantics, which need to be interpretable by the target resource handler. A validator SHOULD NOT reject UCANs with resources that it does not know how to interpret.
+
+If any of the following criteria are not met, the UCAN MUST be considered invalid.
+
+## 5.1 Time
+
+A UCAN's time bounds MUST NOT be considered valid if the current system time is prior to the `nbf` field, or after the `exp` field. This is called "ambient time validity".
+
+All witnesses MUST contain time bounds equal to or wider than the UCAN being delegated to. If the witness expires before the outer UCAN -- or starts after it -- the reader MUST treat the UCAN as invalid. This is called "timely delegation".
+
+A UCAN is valid inclusive from the `nbf` time, and until the `exp` field. If the current time is outside of these bounds, the UCAN MUST be considered invalid. A delegator or invoker SHOULD account for expected clock drift when setting these bounds. This is called "timely invocation".
```

Comment:
Okay, I'll clarify this in the spec. A well-formed delegation MUST have narrower time bounds than its parent, otherwise it's an escalation on time bounds.